docs/gpus.md

```diff
@@ -44,7 +44,10 @@ GPU version is available. To force allreduce to happen on CPU, pass `device_dens
 opt = hvd.DistributedOptimizer(opt, device_dense='/cpu:0')
 ```
 
-### Advanced: Have GPUs and networking with GPUDirect?
+### Advanced: Have GPUs and networking with RDMA and GPUDirect?
+
+**Note**: This section is only relevant if you have RDMA and GPUDirect. Please note that the `nv_peer_memory` driver is
+required for GPUDirect support, and MPI **must** be compiled with CUDA support - otherwise your program will crash.
```

Comment:
Pull "This section is only relevant if you have RDMA and GPUDirect." out of the note (have it be the first text below the heading)

"Please note that the `nv_peer_memory` driver is
required for GPUDirect support, and MPI **must** be compiled with CUDA support - otherwise your program will crash." --> For GPUDirect, Horovod requires the `nv_peer_memory` driver. To prevent your program from crashing, MPI **must** be compiled with CUDA support.